src/styles.css

```diff
@@ -1 +1,14 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Raleway:wght@700&display=swap');
\ No newline at end of file
+@font-face {
```

Comment:
@MyColaAnikeiev , move font styles to separate file (mostly it is placed in src/assets/fonts)

Reply:
You mean place style css file in src/assets/fonts folder and add it's path to angular.json styles array, am i right? Because placing css into assets folder doesn't feel right to me but i could be wrong.